ui/src/wizard/LocationListStep.ts

```diff
@@ -40,11 +40,17 @@ export class LocationListStep<T extends ILocationWizardContextInternal> extends
     }
 
     public static async setLocation<T extends ILocationWizardContextInternal>(wizardContext: T, name: string): Promise<void> {
+        /**
+         * Get location from locations list if passed in default location name is availble
+         * Otherwise get the first location from location list.
+         */
         const locations: SubscriptionModels.Location[] = await LocationListStep.getLocations(wizardContext);
         name = generalizeLocationName(name);
```

Comment:
Do we want to do this only for CustomCloud or even for azure this approach is fine?

Reply:
As far as I am concerned, passed in default location name would be available in Azure. Therefore, I don't think we will get into that if condition in Azure.